### ISSUE TITLE
Question about naming criteria: can a creator’s first name be used?

### ISSUE THREAD

**Issue by arrowtype** [google/fonts#2486]
I’m advising an artist on creating a handwriting-based font which will most likely be OFL, and they hope for it to eventually be on Google Fonts (though I also explained that there is no guarantee, and it takes time to happen when it does, and that I am merely stating my observations and can obviously not speak on behalf of Google Fonts or anyone else). 

However, I have a question about one of the naming criteria:

> The font family name does not include any copyright holder names or registered trademarks, and no initials or abbreviations, and no references to languages or writing systems; it should be a simple and unique name, and a limited but easy way to test for uniqueness is namecheck.fontdata.com

I am interpreting “does not include any copyright holder names or registered trademarks” means that a font cannot be named after a trademarked name (e.g.  “Target Sans” or “Nike Mono” etc) and can’t be someone’s full name (e.g. “Stephen Nixon Serif”). 

However, I assume that a first name would be an acceptable font name, particularly for a font based on handwriting. For example, if someone named Ezekiel Smith (this is a made-up name) made a font based on their handwriting, “Ezekiel” would be a pretty perfect font name on all other criteria: it’s memorable, specific, meaningful, short, and unique. After all, Google Fonts already includes several fonts which are include first names: [Francois One](https://fonts.google.com/specimen/Francois+One), [Yanone Kafeesatz](https://fonts.google.com/specimen/Yanone+Kaffeesatz), [Karla](https://fonts.google.com/specimen/Karla), [Anton](https://fonts.google.com/specimen/Anton). 

However, I don’t want to pass along this recommendation and have it be the reason that this font is later rejected. Might it be possible to clarify that criterion in the spec, or at least clarify it here?

Thank you for any insights!

**Comment by felipesanches** [comment]
I'd also mention Beth Ellen (https://fonts.google.com/specimen/Beth+Ellen) by Rob Jellinski (son of Beth)

**Comment by arrowtype** [comment]
Excellent example, Felipe! Thanks.

**Comment by davelab6** [comment]
Yes, that would be fine. There is also https://fonts.google.com/specimen/Patrick+Hand as a direct example. 

The point is that we want projects to be open to contributions :)
